curvesim/pool_data/__init__.py

```diff
@@ -45,28 +49,34 @@ def get_data_cache(address, chain="mainnet", days=60, end=None):
 
 
 def get_metadata(
```

Comment:
There is a merge conflict as #277  moved `get_metadata` to a new file `curvesim.pool_data.queries.metadata` and then imported it into this file from there